app/controllers/sessions_controller.rb

```diff
@@ -49,14 +51,14 @@ def update_password
     if password_params_valid?
       current_user.update_password(params[:old_password],
                                    params[:new_password1])
-      flash[:notice] = t('flashes.logins.new_password_set')
+      flash[:notice] = t('flashes.sessions.new_password_set')
       redirect_to teams_path
     else
       render :show_update_password
     end
   end
 
-  # POST /login/changelocale
+  # POST /sessions/changelocale
```

Comment:
move to nested controller ?
session/locale#update ? POST /session/locale OR POST /profile/locale
maybe own story for this ?

Reply:
Can be done in the Profilerestyling Story